package.json

```diff
@@ -61,7 +61,7 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
-    "gts": "^1.1.2",
+    "gts": "^2.0.2",
```

Comment:
Upgrading from gts 1.x to gts 2.x requires a number of steps (see https://github.com/google/gts/releases/tag/v2.0.0). I would stick with gts 1.x for the moment and move to gts 2.x in another PR

Reply:
Ok, I think this is the thing that npm audit was barking about. I'll revert it.